test/tools/ossfuzz/SolidityGenerator.h

```diff
@@ -89,7 +94,7 @@ struct TestState
 		sourceUnitPaths.insert(_path);
 		currentSourceUnitPath = _path;
 	}
-	/// @returns true if @name sourceUnitPaths is empty,
+	/// Returns true if @name sourceUnitPaths is empty,
```

Comment:
Most of places it uses `@returns` and here you have changed it, maybe it is better to change other occurrences to be consistent

Reply:
Done.